### DISCUSSION TITLE
An Action that takes you to the parent screen

### DISCUSSION THREAD

**Discussion by naikrovek** [elgatosf/elgato#89] [Ideas]
The "Go to Page" action _almost_ does what I'm asking, but not quite.

If I'm in a folder, I would like to be able to run a Multi Action and have the final action be an action which is equivalent to the "go up" action which shows up in the top left of newly created folders.  I'd like that "go up" button to be an action that I can add to an arbitrary multi action that I create.  I don't want to have to specify the destination (this is so I can move folders around inside other folders or wherever without needing to modify the multi actions contained within.)

**Comment by GeekyEggo** [comment]
Hi @naikrovek, great news - with the release of Stream Deck 6.5 we're introducing a new feature, "Folder Auto Exit".
> You can now instantly exit a folder after executing an action with Auto Exit. This is great for folders containing actions like sound clips or temporary overlays. Set the time delay from immediately to 1 minute. This setting can be applied to entire folders or individual actions.

Stream Deck 6.5 is not yet being automatically rolled out, however it is available on [our downloads page](https://www.elgato.com/uk/en/s/downloads).

[Auto_Folder_Exit.webm](https://github.com/elgatosf/elgato/assets/1429781/e4f1c3da-8e69-43f4-90df-8eee19aaeb47)
